doc/release-notes-14565.md

```diff
@@ -0,0 +1,5 @@
+Low-level RPC changes
+---------------------
+
+The `importmulti` RPC will now contain a new `warnings` field with strings
```

Comment:
nit, could say:
 - it's per request result
 - it's not present if there are no warnings

Reply:
Done.